core/src/main/resources/error/error-classes.json

```diff
@@ -797,6 +797,11 @@
     ],
     "sqlState" : "42602"
   },
+  "INVALID_IDENTIFIER_HAS_MORE_THAN_2_NAME_PARTS" : {
```

Comment:
Let's only use string for error class name.

BTW, maybe we can integrate it with `INVALID_IDENTIFIER` and reuse it ??

Or maybe `IDENTIFIER_TOO_MANY_NAME_PARTS` similar to `TEMP_VIEW_NAME_TOO_MANY_NAME_PARTS` ?

Reply:
Invalid identifier i used for bad characters IIRC.
IDENTIFIER_TOO_MANY_NAME_PARTS is fine.

Reply:
Done